addons/sourcemod/scripting/shop/player_manager.sp

```diff
@@ -635,6 +636,32 @@ public int PlayerManager_ToggleClientCategoryOff(Handle plugin, int numParams)
 	h_KvClientItems[client].Rewind();
 }
 
+public int PlayerManager_GetClientItems(Handle plugin, int numParams)
+{
+	int client = GetNativeCell(1);
+	
+	char error[64];
+	if (!CheckClient(client, error, sizeof(error)))
+		ThrowNativeError(SP_ERROR_NATIVE, error);
+
+	ArrayList hArrayItems = new ArrayList(ByteCountToCells(4));
+	KeyValues hKVItems = h_KvClientItems[client];
```

Comment:
it's just for convenience, you exactly don't need to close it ?

Reply:
Close what? `hKVItems`? I just copy pointer for easy reading.
If i close `hKVItems`, `h_KvClientItems[client]` will be closed too.